ssaHAL/ssa/core.py

```
@@ -42,6 +43,24 @@ def __fw_update_callback(_ssa: SSA, update_str: str):
+    if prop not in ssa.__properties:
+        print(f"[ERROR] Property `{prop}` does not exist. Create it using `create_property` before setting it.")
+        return
+
+    value = json.loads(msg)
+    ssa.set_property(prop, value)
+
+class ActDictElement():
+    def __init__(self,
+                 callback: Optional[Callable[['SSA', str, ...], None]] = None,
+                 node_name: Optional[str] = None,
+                 children: Optional[Dict[str, ActDictElement]] = None):
```

Ruff (0.8.2)

Undefined name `ActDictElement`

(F821)